package.json

```diff
@@ -6,7 +6,12 @@
   "version": "0.8.0",
   "scripts": {
     "start": "react-native start",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "compile:test-ios": "PLATFORM=ios babel-node cli/compile.js --presets react-native",
```

Comment:
Although now that I see that the tests are using async/await and generators, we probably still need this?